generator/statistics.cpp

```diff
@@ -58,7 +56,7 @@ namespace stats
 
     void operator() (FeatureType & f, uint32_t)
     {
-      f.ParseBeforeStatistic();
+      f.ParseHeader2();
```

Comment:
The current name is bad, let's think about a better one.

Reply:
yeah its because there is a very first one-byte header also

Reply:
ParseSecondHeader?

Reply:
Well, ParseHeader2 name is ok, while this function is in private sections as it was before.
Why not keep previous ParseToCalculateStats() ?

Reply:
Or FeatureType::GetInnerStats() can call it inside.

Reply:
> Well, ParseHeader2 name is ok, while this function is in private sections as it was before. Why not keep previous ParseToCalculateStats() ?

Because it was used in a context that has nothing to do with stats, i.e. `feature_segments_checker.cpp` and `feature_list.cpp`.